### PR TITLE
#122 remove unused crate serde

### LINKED ISSUE #122: hate: warning
```text
warning: unused extern crate
  --> hate/src/lib.rs:20:1
   |
20 | extern crate serde;
   | ^^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_extern_crates)] on by default
```

### PR DESCRIPTION
**Pull request by hack3p**
Fix #122

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
+# IDE files
```
Comment (ozkriff) [inline .gitignore]: It's usually better to ignore IDE-related files in a personal global `.gitignore` file - https://robots.thoughtbot.com/global-gitignore .

Could you please revert this change? :)

Comment (hack3p) [reply .gitignore]: Yes, i can do this. However many project use internal gitignore, like [servo](https://github.com/servo/servo/blob/master/.gitignore#L27). 
How do you do it? 
a. revert -> fix -> commit -> push
b. fix -> commit -> push 
or no difference?

Comment (ozkriff) [reply .gitignore]: > However many project use internal gitignore, like servo.

Yep, not everybody agrees on this :(

> How do you do it?

It's small fix, so whatever you're comfortable with :)

Comment (hack3p) [reply .gitignore]: Fixed

**Comment by ozkriff** [comment]
Thanks, man! :-)
